### PR TITLE
remove WIP annotations

### PR DESCRIPTION
**Pull request by zirain**
xref: https://github.com/istio/istio/issues/51611

### CODE REVIEW

**Review by hzxuzhonghu — APPROVED** [review]
These annotations may be changed

File: annotation/annotations.yaml [hunk 1]
```diff
@@ -537,7 +537,7 @@ annotations:
       (optional). The wildcard character '*' can be used to redirect all outbound traffic.
       An empty list will disable all outbound redirection.
     deprecated: false
-    hidden: false
+    hidden: true
```
Comment (howardjohn) [inline annotation/annotations.yaml:540]: why hidden? this has been implemented and used for a long time

Comment (howardjohn) [reply annotation/annotations.yaml:540]: Oh it doesn't have `sidecar` in it...

I would just revert these entirely. I think they were accidentally merged (I was the approver) thinking they were just adding the sidecar ones

Comment (zirain) [reply annotation/annotations.yaml]: remove them first, we should add something to point out it related to ambient or ztunnel.

**Review by howardjohn — APPROVED** [review]
ty

**Pull request merged**